packages/react-integration/demo-app-ts/src/components/demos/DonutUtilizationChartDemo/DonutUtilizationSimpleDemo.tsx

```diff
@@ -12,7 +12,7 @@ export class DonutUtilizationSimpleDemo extends React.Component {
         <div style={{ backgroundColor: 'white', height: '230px', width: '500px' }}>
           <ChartDonutUtilization
             data={{ x: 'GBps capacity', y: 75 }}
-            labels={({ datum }) => (datum.x ? `${datum.x} - ${datum.y}%` : null)}
+            labels={({ datum }) => (datum.x ? `${datum.x} - ${datum.y}%` : '')}
```

Comment:
Poking around and found that if we swap the type for [labels here](https://github.com/patternfly/patternfly-react/blob/v4/packages/react-charts/src/components/ChartDonutUtilization/ChartDonutUtilization.tsx#L326) to be `labels?: string[] | ((data: any) => string | null);` then users can specify `null` as a value without throwing errors and without showing the empty tooltip box.

WDYT? @dlabrecq would this cause any problems you're aware of?

Reply:
My concern is that it's not the only place where `null` and `undefined` are valid. Yes, we could change this one prop, but it doesn't address any other (pass-thru) Victory props. The demo is only touching a single point here -- we're not testing anything else.

Does this mean all strings cannot be null, unless we explicitly add null to the type? This would be contrary to the types defined by @types/victory. 

Curious what happens when we use VictoryPie directly, instead of the ChartPie component?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/victory/index.d.ts#L1550

Reply:
My concern is that it's not the only place where `null` and `undefined` are valid. Yes, we could change this one prop, but it doesn't address any other (pass-thru) Victory props. The demo is only touching a single point here -- we're not testing anything else.

Does this mean all strings cannot be null, unless we explicitly add null to the type? This would be contrary to the types defined by @types/victory. 

Curious what we would do if using VictoryPie directly, instead of the underlying ChartPie component? It appears that VictoryPie would have the same issue with the demo's strict rules.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/victory/index.d.ts#L1550

Reply:
> Does this mean all strings cannot be null, unless we explicitly add null to the type?

This is a great question, and it's exactly the understanding that I was _starting_ to formulate, but wanted to verify. That does seem like what this setting is geared toward enforcing.

> This would be contrary to the types defined by @types/victory

Yea, and while we do this in other places (create our own abstraction on third party lib props), I guess we've treated Victory/charts stuff differently to this point to keep things simple/pass-through in terms of api and documentation.

> it doesn't address any other (pass-thru) Victory props

What's an example of this? If we need to support this, then we could add to the integration app an example that exercises this code path. Is it something that's easily illustrated?

Reply:
There are many Victory-based props that can accept null or undefined. Unfortunately, I do not know all the use-cases, but anything related to data. My point is that enabling strict mode may cause some breakage we haven't anticipated.

Reply:
I see, thanks for explaining. I knew it would be a potentially challenging thing to achieve. It does raise good discussion though. If the incoming Victory types don't work out of box in strict mode, seems like we have the options of either continuing to expose these props as-is and for PF to not support this specific strict mode for chart components, or adjust our types as necessary and possibly reduce our support of the Victory props to the subset that we feel is worth supporting and relevant to PF needs.

Victory has a large API surface area, so just brainstorming to see how we can best align it with what we're supporting on our end.

Is there another path forward I'm not thinking of? Otherwise, let me know what you think :)

Reply:
Omitting Victory props probably isn't a good option. Perhaps we should discuss offline, along with the infrastructure team?